src/marketing/products-viewed-compared.md

```diff
@@ -12,13 +12,19 @@ The Recently Viewed and Recently Compared blocks usually appear in the right sid
 
 1. Expand ![Expansion selector]({% link images/images/btn-expand.png %}) the **Recently Viewed/Compared Products** section and do the following:
 
-    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-compared-products.png %}){: .zoom}
+    ![Catalog configuration - recently viewed/compared products]({% link images/images/config-catalog-catalog-recently-viewed-and-compared-products.png %}){: .zoom}
     [_Recently Viewed/Compared Products_]({% link configuration/catalog/catalog.md %})
 
+   - Set **Synchronize widget products with backend storage** to synchronize product widget information such as product id with your current product storage availability in database. In order to reuse this info on different devices.
+
    - Set **Show for Current** to the website, store, or store view where the configuration applies.
 
    - In the **Default Recently Viewed Products Count** field, enter the number of recently viewed products to appear in the list.
 
    - In the **Default Recently Compared Products Count**, enter the number of recently compared products to appear in the list.
```

Comment:
I n=know you did not update this one, but let's fix while we are here.

```suggestion
   - For **Default Recently Compared Products Count**, enter the number of recently compared products to appear in the list.
```

Reply:
Done :)